tests/subsys/logging/log_stress/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(log_core)
```

minor nit.
Should we call the project `log_stress` ?
```suggestion
project(log_stress)
```